wiki/Tournaments/README.md

```diff
@@ -1,9 +1,33 @@
-# Community Tournaments
+# Tournaments
 
-This is the place where tournament hosts can add a wiki page for their tournaments.
+This is where wiki articles for tournaments live, both official and community-run. You are welcome to add new tournaments here, but you should have approval from the tournaments' host(s) before doing so.
 
-## Contributing
+Read the ["contributing" file](/CONTRIBUTING.md) if you're a new contributor to osu-wiki. If you have any questions about tournament articles, ask in the `#osu-wiki` channel of the [osu!dev Discord server](https://discord.gg/ppy) or [open an issue](https://github.com/ppy/osu-wiki/issues/new).
 
-We urge you to look at the [template for official tournaments](Template) as a reference for how an acceptable tournament wiki page should look.
+## Adding a new tournament
 
-Please make sure to read the README on the [front page of this repository](https://github.com/ppy/osu-wiki) for other contributing guidelines.
+It's usually the easiest to start from the [tournament template](TEMPLATE.md) if you're adding a new tournament:
+
+1. Create a new directory in the `Tournaments` folder for the tournament or tournament series. The directory name should be the acronym or short name of the tournament. If the tournament is a series, create another directory under it named after the tournament's iteration in the series.
+
+   Examples:
+
+   - osu! World Cup 2014 is in the `OWC/2014` directory
+   - Catch French Cup 2 is in the `CFC/2` directory
+
+2. Copy `TEMPLATE.md` to the tournament's directory, and rename it to `en.md`. If you are adding a new tournament series, copy `TEMPLATE-SERIES.md` into the series' directory, and rename it to `en.md` as well.
+
+3. Fill in the placeholders placed in parenthesis in each of the new files with the tournament's information.
+
+   Examples:
+
+   - `{TITLE}` - title of the tournament (osu! World Cup 2014)
+   - `{SHORT_TITLE}` - acronym or short name of the tournament (OWC 2014)
+   - `{SERIES_TITLE}` - title of the tournament series (osu! World Cup)
+   - `{ITERATION_DIRECTORY}` - directory name of the iteration (2014)
+
+   Double-check the grammar and wording around placeholders; the tournament may require some changes depending on how the title is written.
```

Comment:
```suggestion
   Double-check the grammar and wording around placeholders, as it may require some changes depending on how the title is written.
```